### ISSUE TITLE
[issue] Install Error 

### ISSUE THREAD

**Issue by kdh4win4** [ramnathv/rCharts#628]
I have tried more than a dozen times however, I could not install this packages on my Mac.
Please help me code correctly. Dr. Ramnathv

I installed devtools and other packages for rCharts.("plyr, RCurl, RJSONIO, whisker, yaml"))
However, it is not working at all. Thank you. 

> install_github("rCharts", "ramnathv") 
Downloading github repo ramnathv/rCharts@master
Error in function (type, msg, asError = TRUE)  : 
  Failed connect to api.github.com:443; Connection refused
In addition: Warning message:
Username parameter is deprecated. Please use ramnathv/rCharts 

> install_github("ramnathv/rCharts")
Downloading github repo ramnathv/rCharts@master
Error in function (type, msg, asError = TRUE)  : 
  Failed connect to api.github.com:443; Connection refused

**Comment by ramnathv** [comment]
This seems to be an issue with your internet connection. Are you behind a firewall?

**Comment by kdh4win4** [comment]
no. Sir . the firewall is off. I have tried this at home and work place. 
I did this work in R 3.2 and R 2.6.5 however all failed 
I uninstall R 3.2 and install R 3.1.3 for installation for rCharts temporarily. 
Moreover, I downlaoded below file however it is also not working. 
I need your help. Dr. Ramnathv. Thank you. 
install.packages("downloader")
library(downloader)
 download("https://github.com/ramnathv/rCharts/archive/master.tar.gz", "rCharts.tar.gz")
install.packages("rCharts-master(1).tar.gz", repos = NULL, type = "source")
